ge/org.osate.ge/src/org/osate/ge/internal/ui/util/ContextHelpUtil.java

```diff
@@ -26,26 +26,84 @@
 import org.eclipse.swt.widgets.Control;
 import org.eclipse.ui.PlatformUI;
 
-public class ContextHelpUtil {
+/**
+ * Utility help related to context sensitive help
```

Comment:
I am making some changes myself like you mentioned, but wanted your opinion on this.  Is this sentence written as intended or should it be something like Utility "class" related to context sensitive help?

Reply:
Yes, it should be changed. `Utility class related to context sensitive help` would be fine.